e2e/utils/fixture.ts

```diff
@@ -22,7 +23,7 @@ interface FixtureCreateOptions {
   forceDeterministicTerminalOutput?: boolean;
 }
 
-type RunCommandResult = { stdout: string; stderr: string; combinedOutput: string };
+export type RunCommandResult = { stdout: string; stderr: string; combinedOutput: string };
```

Comment:
How come this was exported as part of the changes, I don't think it is used outside this file?

Reply:
RunCommandResult is returned by public functions of `Fixture`, so it should be exported too. I don't think it ever makes sense to allow the consuming code to obtain an instance of a type but have that type be private/not exported. 

This way, the consuming e2e tests can explicitly declare this type:
```
let commandOutput: RunCommandResult;
```

Reply:
removed the exports